pkg/cli/scan.go

```diff
@@ -416,32 +415,12 @@ func (p *scanParams) resolveInputsToScan(ctx context.Context, args []string) (in
 			fmt.Println("📡 Finding remote packages")
 		}
 
-		for _, arg := range args {
-			targetPaths, cleanup, err := resolveInputForRemoteTarget(ctx, arg)
-			if err != nil {
-				return nil, nil, fmt.Errorf("failed to resolve input %q for remote scanning: %w", arg, err)
-			}
-			inputs = append(inputs, targetPaths...)
-			cleanupFuncs = append(cleanupFuncs, cleanup)
-		}
+		return resolveInputsForRemoteTarget(ctx, args)
 
 	default:
 		inputs = args
 	}
 
-	cleanup = func() error {
```

Comment:
nit: Since we know we're not setting the value of `cleanup` any more, I think it'd be more clear now if the `return` line below used `nil` in place of `cleanup`